dcos/mesos.py

```diff
@@ -51,8 +51,11 @@ def get_dcos_url(self, path):
         """
         if self._dcos_url:
             return urllib.parse.urljoin(self._dcos_url, path)
+        elif self._mesos_master_url:
+            mesos_master_url = ':'.join(self._mesos_master_url.split(':')[:2])
+            return urllib.parse.urljoin(mesos_master_url + '/', path)
```

Comment:
Agreed this would be the best fix: the documentation in both the Github page and DCOS specify that `dcos_url` has to be specified. Also, I'm pretty sure there's multiple places in the code that require that specific variable.

However, I don't know if there's any implications of doing that. If you guys can confirm that there's no issue, I can remove the if and elif statements.